webapp/views.py

```diff
@@ -82,7 +82,21 @@ def releasenotes_redirect():
     Old apache redirects: https://pastebin.canonical.com/p/3TXyyNkWkg/
     """
 
-    ver = flask.request.args.get("ver")
+    releaseName = {
+        "12.04": "PrecisePangolin",
+        "12.10": "QuantalQuetzal",
+        "13.04": "RaringRingtail",
+        "13.10": "SaucySalamander",
+        "14.04": "TrustyTahr",
+        "14.10": "UtopicUnicorn",
+        "15.04": "VividVervet",
+        "15.10": "WilyWerewolf",
+    }
+
+    ver = flask.request.args.get("ver")[:5]
+
+    if releaseName[ver]:
```

Comment:
This fails if `ver` isn't in the dictionary.

![image](https://user-images.githubusercontent.com/519935/78540148-9c359a80-77eb-11ea-95c3-b18948bcb0d4.png)

What you want is `if ver in releaseName:`